src/components/Atoms/EventsButton/EventsButton.stories.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
```

Comment:
React is warning about a unique key prop for this story.

Reply:
I have seen this, but I have trouble reproducing it. It only seems to do it in Storybook and only sometimes (refreshing the browser). Do you have a suggestion of what would need a key?